.github/workflows/ci.yml

```diff
@@ -26,4 +26,6 @@ jobs:
     - name: Install Dependencies
       run: npm install
     - name: Run
-      run: npm run ${{ matrix.version }}
```

Comment:
It made command npm run , but returned code is success 0.
https://github.com/namikingsoft/portfolio-via-headless-cms/actions/runs/3552629832/jobs/5967657074